storage/transfermanager/downloader.go

```diff
@@ -31,6 +33,12 @@ import (
 	"google.golang.org/api/iterator"
 )
 
+// maxChecksumZeroArraySize is the maximum amount of memory to allocate for
+// updating the checksum. A larger size will occupy more memory but will require
+// more updates when computing the crc32c of a full object.
```

Comment:
This seems wrong, I would imagine larger means fewer updates?

Reply:
Correct, I messed up that wording.